ui/tests/integration/components/replication-secondary-card-test.js

```diff
@@ -0,0 +1,99 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+const REPLICATION_DETAILS = {
+  state: 'stream-wals',
+  connection_state: 'ready',
+  lastWAL: 0,
+  lastRemoteWAL: 10,
+  delta: 10,
+};
+
+const STATE_ERROR = {
+  state: 'idle',
+  connection_state: 'ready',
+  lastWAL: 0,
+  lastRemoteWAL: 10,
+  delta: 10,
+};
+
+const CONNECTION_ERROR = {
+  state: 'stream-wals',
+  connection_state: 'transient_failure',
+  lastWAL: 0,
+  lastRemoteWAL: 10,
+  delta: 10,
+};
+
+module('Integration | Enterprise | Component | replication-secondary-card', function(hooks) {
+  setupRenderingTest(hooks);
+  const title = 'States';
+  const hasErrorClass = false;
```

Comment:
nit: these 2 consts should be moved above, just like CONNECTION_ERROR and STATE_ERROR. it keeps all our consts nice 'n tidy!